components/service-operator/api/v1beta1/postgres_types.go

```diff
@@ -24,6 +24,7 @@ import (
 
 // AWS allows specifying configuration for the Postgres RDS instance
 type AWS struct {
+	DiskSizeGB   int    `json:"diskSizeGB,omitempty"`
```

Comment:
I'd expect to see a corresponding CRD update...

Reply:
What do you mean by this? The example?

Reply:
https://github.com/alphagov/gsp/blob/postgres_operator_add_disk_size_as_a_configurable_option/components/service-operator/config/crd/bases/database.gsp.k8s.io/v1beta1_postgres.yaml